src/components/entries/TextField.js

```diff
@@ -23,9 +28,12 @@ function Textfield(props) {
     label,
     onInput,
     feel = false,
-    value = ''
+    value = '',
+    show = noop
```

Comment:
So every entry (e.g. property entry) has to define on its own whether it shall be shown or not? What would such a callback look like? Given the [referenced branch in bpmn-js-properties-panel](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/601/files#diff-7650658698a45a7549c8cdba125ffa207d194a6be03101347fb1bf88b62d6b5aR138) I see that this could mean quite an effort. Will there be an easier way to provide such a callback?

Reply:
First of all, providing a `show` callback is optional so not every entry has to implement one. The generic callback is necessary because it cannot be assumed e.g. that every entry is shown based on an ID or a path. To make things easier we can create utilities in bpmn-js-properties-panel to make creating `show` callbacks less repetitive (e.g. `function showPath(id, path) { ... }`).

Reply:
>  To make things easier we can create utilities in bpmn-js-properties-panel to make creating show callbacks less repetitive (e.g. function showPath(id, path) { ... }).

👍